### DISCUSSION TITLE
Sorting "My Tones" and Setlists

### DISCUSSION THREAD

**Discussion by crillev** [soundshed/soundshed-app#25] [Ideas]
Hi,
It sure would be neat to be able to sort or move presets within "My Tones".
It would also be neat to be able to make my own "Setlists".

Otherwise, really great app!
Cheers / Chris from Sweden

**Comment by webprofusion-chrisc** [comment]
Absolutely, setlist and other custom grouping is planned. Regarding sorting and moving favourites - I can see alphabetical or by genre or even by date added but would you like to arbitrarily drag them into a specific order? Would new ones just go on the end of the list?

**Reply by crillev** [reply]
Well, since Setlists are up high on my priority list and you need to be able to put presets in a certain order, I guess you automatically also would have to solve the possibilty to drag the presets around in any kind of list.
My priority would be:
1. Setlists
2. Group by genre
3. Group by artist
4. Long alphabetical complete list

New entrys should automatically go to the appropriate place in the alphabetical list AND (if possible) be given the option to choose Setlists, genre, artist, etc. at the time of saving the preset, where it would be placed last. 
The exact placement would be chosen manually after it had been placed last in a chosen list.
/Chris

**Reply by crillev** [reply]
Forgot to mention, but if you need/want a beta-tester, I'd be happy to oblige! :-)

**Reply by webprofusion-chrisc** [reply]
Thanks, I'm thinking only setlist/custom lists would have a draggable order, as that way each list can have it's own custom order.

**Reply by crillev** [reply]
Yeah, that makes sense!
I got carried away there!
